pkg/ebpf/tracer.go

```diff
@@ -90,8 +90,8 @@ const (
 
 func NewTracer(config *Config) (*Tracer, error) {
 	// make sure debugfs is mounted
-	if mounted, msg := util.IsDebugfsMounted(); !mounted {
-		return nil, fmt.Errorf("%s: %s", "system-probe unsupported", msg)
+	if mounted, err := kernel.IsDebugFSMounted(); !mounted {
+		return nil, fmt.Errorf("%s: %s", "system-probe unsupported", err)
```

Comment:
just to verify: will this continue to print out the instructions for mounting debugfs ?

Reply:
Correct.